app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t "title_page"
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
a so sánh giúp e: nil?, empty?, blank?, present? và ở đây nên dùng cái nào nhe

Reply:
nil: nếu đối tượng là nil nó sẽ trả về giá trị là TRUE còn lại nó trả về FALSE
empty: kiểm tra độ dài đối tượng có bằng 0 hay không. Chủ yếu dùng cho String, Array, Hash
blank: kiểm tra những đối tượng nil, false, empty, chuỗi là khoảng trắng và trả về giá trị là TRUE
present: Ngược lại với blank?
Và ở đây ta nên dùng blank? vì nếu giá trị chuyền vào là chuỗi các khoảng trắng thì empty? nó sẽ trả về False và blank? nó sẽ trả về True mà ở đây ta cần giá trị là True để không in ra khoảng trắng đó.